components/pkg-export-docker/defaults/Dockerfile.hbs

```diff
@@ -1,7 +1,10 @@
 FROM scratch
 ENV PATH {{path}}
+{{install_hook_env}}
+ENV OFFLINE_INSTALL ON
```

Comment:
yes. If the user has toggled the feature, then it gets passed into the docker build context so the install hook will run.

Reply:
I was referring to the `ENV OFFLINE_INSTALL ON` line... that's being unconditionally added here, and I was surprised, since nothing else in this PR seems to be concerned with offline installation.

Reply:
oh I added that because the whole point of the `hab pkg install` was to kick off the install hooks ignored setting up the rootfs. I set `ENV OFFLINE_INSTALL ON` to eliminate any possibility of actually downloading new stuff from anywhere.

Taking up your thoughts on leveraging handlebar conditinals, this will change so that it will only be set if the INSTALL_HOOK feature is set.